### PR TITLE
Fix Edit Wiki Page without previous revision ID

### PR DESCRIPTION
**Pull request by CrustyJew**
Sending in "previous" with null causes a 409 response from Reddit.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Your indentation is wrong. Looks fine otherwise.

File: RedditSharp/Wiki.cs [hunk 1]
```diff
@@ -102,14 +102,21 @@ public Listing<Post> GetPageDiscussions(string page)
...
+				page = page,
+				uh = Reddit.User.Modhash
+			};
+			List<string> addParams = new List<string>();
+			if ( previous != null ) {
+				addParams.Add( "previous" );
+				addParams.Add(previous);
+			}
+			if ( reason != null ) {
+				addParams.Add( "reason" );
```
Comment (SirCmpwn) [inline RedditSharp/Wiki.cs]: Extra spaces here is not correct, either.

**Comment by CrustyJew** [comment]
I can safely say I have no idea what happened to that as it looks fine in VS for me. I'm guessing I broke some line ending crap, let me fix it real quick

**Comment by SirCmpwn** [comment]
You used tabs when you should have used spaces. And you added extra spaces inside of your parenthesis.

**Comment by SirTony** [comment]
Brace style is also inconsistent with the rest of the library.

**Comment by SirCmpwn** [comment]
Right, that too.

**Comment by CrustyJew** [comment]
Apparently Visual Studio lost all it's settings and inserted tabs (messed up the indents) instead of spaces, and bracing was off. Anyway, I think that fixed it.

**Comment by SirCmpwn** [comment]
Looks better, but there's still the spaces inside of your parenthesis.

**Comment by CrustyJew** [comment]
Fixed spacing and squashed commits

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**